federation/cluster/common.sh

```diff
@@ -135,6 +135,30 @@ function create-federation-api-objects {
 	    fi
 	    sleep 5
 	done
+
+	# Poll until DNS record for federation-apiserver is propagated
+	# TODO(colhom): REQUIREMENT for any other providers which use a DNS name for routing to loadbalancers
+	# right now it's just AWS that does this.
+	which nslookup
+	set +o errexit
+	if [[ "$KUBERNETES_PROVIDER" == "aws" ]];then
```

Comment:
Yes. We dont need to run nslookup for GCE, but I would like to to remove that if condition if we can. Nothing wrong in running nslookup for GCE IP address?